src/FFNetAgent.hpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2015 - 2023, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef FFNETAGENT_HPP_INCLUDE
+#define FFNETAGENT_HPP_INCLUDE
+
+#include <vector>
+#include <string>
```

Comment:
```suggestion
#include <string>
#include <cstdint>
#include <utility>
```
For `uint64_t` use and `std::pair` use.